data/goal_src/jak1/engine/mods/mods-common.gc

```diff
@@ -8,142 +8,140 @@
+  (begin
+    (close-specific-task! (game-task lavatube-start) (task-status need-reward-speech))
+  ))
+  )
 
 (defun skip-samos-intro ()
-;;Skip the samos cutscenes if the randomizer is on. Cutscenes bad.
-(close-specific-task! (game-task intro) (task-status need-resolution))
-(close-specific-task! (game-task beach-ecorocks) (task-status need-introduction))
-)
+  ;; Skip the samos cutscenes if the randomizer is on. Cutscenes bad.
+  (close-specific-task! (game-task intro) (task-status need-resolution))
+  (close-specific-task! (game-task beach-ecorocks) (task-status need-introduction))
+  )
 
 (defun change-geyser-spawn-game-start ()
-(if (string= (symbol->string (-> (level-get-target-inside *level*) name)) "training")
-(set! (-> *game-info* current-continue) (the-as continue-point (get-continue-by-name *game-info* "game-start")))
-(none)))
+  (if (string= (symbol->string (-> (level-get-target-inside *level*) name)) "training")
+  (set! (-> *game-info* current-continue) (the-as continue-point (get-continue-by-name *game-info* "game-start")))
+  (none))
+  )
 
 
 (defun mods-common-run-each-frame ()
-;;Update global variables
-(set! racer? (aif *target* (case (-> it next-state name) (
-                                      ('target-racing
-                                       'target-racing-bounce
-                                       'target-racing-death
-                                       'target-racing-falling
-                                       'target-racing-grab
-                                       'target-racing-hit
-                                       'target-racing-jump
-                                       'target-racing-smack
-                                       'target-racing-start) #t))))
-(set! flutflut? (aif *target* (case (-> it next-state name) (
-                                      ('target-flut-air-attack
-                                       'target-flut-air-attack-hit-ground
-                                       'target-flut-double-jump
-                                       'target-flut-falling
-                                       'target-flut-grab
-                                       'target-flut-hit
-                                       'target-flut-hit-ground
-                                       'target-flut-jump
-                                       'target-flut-running-attack
-                                       'target-flut-stance
-                                       'target-flut-start
-                                       'target-flut-walk) #t))))
-;;end
-
-;;Enable/disable things
-(if (enable-unlockZoomers-20-cells)
-(unlockZoomers-20-cells))
-
-(if (enable-skip-samos-intro)
-(skip-samos-intro))
-
-(if (enable-geyser-spawn-game-start)
-(change-geyser-spawn-game-start))
-
-(if (enable-unlock-citadel-door-no-cells)
-(begin
-(close-specific-task! (game-task citadel-sage-green) (task-status need-resolution))
```

Comment:
not needed, users can skip cutscene if they want. if we really feel strongly about this, it should close `need-hint` rather than `need-resolution`